src/codecs/timidity/playmidi.c

```diff
@@ -782,9 +782,11 @@ int Timidity_PlaySome(MidiSong *song, void *stream, Sint32 len)
   return samples * bytes_per_sample;
 }
 
-void Timidity_SetVolume(MidiSong *song, int volume)
+void Timidity_SetVolume(MidiSong *song, float fvolume)
 {
   int i;
+  int volume = (int)(fvolume / 128);
```

Comment:
int volume went from 0 - 128 before so should use fvolume * 128, not fvolume / 128

Reply:
Great point! Thanks for pointing out.

Reply:
Great point! Thanks for pointing out.